client/src/hooks/networks/index.ts

```diff
@@ -0,0 +1,63 @@
+import { useGetOrganizations } from '@/types/generated/organization';
+import { useGetProjects } from '@/types/generated/project';
+import {
+  ProjectListResponseDataItem,
+  OrganizationListResponseDataItem,
+} from '@/types/generated/strapi.schemas';
+
+export type NetworkResponse = {
+  networks: OrganizationListResponseDataItem[] | ProjectListResponseDataItem[];
+  isFetching: boolean;
+  isFetched: boolean;
+  isPlaceholderData: boolean;
+  isError: boolean;
+};
+
+export const useNetworks = ({ page = 1 }: { page: number }) => {
+  const {
+    data: organizationsData,
+    isFetching: organizationIsFetching,
+    isFetched: organizationIsFetched,
+    isPlaceholderData: organizationIsPlaceholderData,
+    isError: organizationIsError,
+  } = useGetOrganizations({
+    populate: '*',
+    'pagination[page]': page,
+    'pagination[pageSize]': 5,
+    sort: 'name:asc',
+  });
+
+  const {
+    data: projectsData,
+    isFetching: projectsIsFetching,
+    isFetched: projectsIsFetched,
+    isPlaceholderData: projectsIsPlaceholderData,
+    isError: projectsIsError,
+  } = useGetProjects({
+    populate: '*',
+    'pagination[page]': page,
+    'pagination[pageSize]': 5,
+    sort: 'name:asc',
+  });
+
+  const networks = [
```

Comment:
Do you think we could sort them all alphabetically?

As we discussed this week, this is not perfect but I think it would look more natural than having the first 5 networks to be organisations and then 5 projects.